connectors/uipath/element-templates/uipath-prototype-add-queue-item.json

```diff
@@ -0,0 +1,392 @@
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Request Body",
+      "description": "JSON payload to send with the request. Please, replace values with curls (e.g. '{{deferDate}}') with real values ('2022-12-28'). Hints: queueName is the only mandatory field; specificContent should have json format; priority should have one of the values: 'Low', 'Normal' or 'High' ",
+      "group": "input",
+      "type": "String",
+      "feel": "required",
+      "value": "={\n\t\"itemData\": {\n\t\t\"DeferDate\":\"{{deferDate}}\", \n\t\t\"DueDate\":\"{{dueDate}}\",\n\t\t\"Priority\":\"{{priority}}\",\n\t\t\"Name\":\"{{queueName0}}\",\n\t\t\"SpecificContent\":\"{{specificContent}}\",\n\t\t\"Reference\":\"{{reference}}\" \n\t}\n}",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "body"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "method",
+        "oneOf": [
+          "post",
+          "put",
+          "patch",
+          "delete"
+        ]
+      }
+    },
+    {
+      "label": "Bearer Token",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.token"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "bearer"
+      }
+    },
+    {
+      "label": "OAuth Token Endpoint",
+      "description": "The OAuth token endpoint",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.oauthTokenEndpoint"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "oauth-client-credentials-flow"
+      }
+    },
+    {
+      "label": "Client ID",
+      "description": "Your application's Client ID from the OAuth client",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.clientId"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "oauth-client-credentials-flow"
+      }
+    },
+    {
+      "label": "Client secret",
+      "description": "Your application's Client secret from the OAuth client",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.clientSecret"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "oauth-client-credentials-flow"
+      }
+    },
+    {
+      "label": "Scopes",
```

Comment:
can we prefill this with the needed scope and hide it?

Reply:
Done!